tests/generic_config_updater/test_incremental_qos.py

```diff
@@ -241,11 +241,15 @@ def test_incremental_qos_config_updates(duthost, tbinfo, ensure_dut_readiness, c
         output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
         if op == "replace" and not field_value:
             expect_op_failure(output)
```

Comment:
For 'replace' and no buffer_pool in the config_db, could you add a log message that buffer_pool and the field value doesn't exist. Would be easy for debugging

Reply:
Added